app/decorators/exercise_decorator.rb

```diff
@@ -15,5 +16,19 @@ def difficulty_level
       "easy"
     end
   end
-  
+
+  def code_to_refactor
+    file_content.split("#SOLUTION")[0].strip
+  end
+
+  def solution
+    file_content.split("#SOLUTION")[1].strip
```

Comment:
Hmm I wonder if there is a better way to organize the data, because index 0 and 1 is unclear in terms of what data you're pulling. Is it possible to have methods that do this and explain what you're getting or at least have keys in a hash that might be more explicit?